lib/api/2.0/nodes.js

```diff
@@ -73,7 +73,7 @@ var nodesGetActiveWorkflowById = controller({ send204OnEmpty: true }, function(r
     return nodes.getActiveNodeWorkflowById(req.swagger.params.identifier.value);
 });
 
-var nodesDelActiveWorkflowById = controller({success: 204}, function(req) {
+var nodesDelActiveWorkflowById = controller({success: 202}, function(req) {
     return nodes.delActiveWorkflowById(req.swagger.params.identifier.value);
 });
```

Comment:
Might be even cooler to:
return nodes.delActiveWorkflowById(req.swagger.params.identifier.value)
.then(function(graph) {
    res.setHeader('Location', '/api/2.0/workflows/' + graph);  
});

(assuming that is the right uri to get the ultimate status of the graph?)